build/common.targets

```diff
@@ -258,7 +258,8 @@
       <!-- remove all net45 assemblies, otherwise loc tools have problem loading two assemblies of same name in same appdomain. net45 doesn't ship in VSIX, so doesn't need loc. -->
       <DllsToLocalizeWithMetadata Include="@(BuiltProjectOutputGroupOutput->'%(FinalOutputPath)')"
 Condition=" ('%(Extension)' == '.dll' OR '%(Filename)' == 'NuGet.CommandLine.XPlat' OR '%(Filename)' == 'NuGet') AND !($([System.String]::new('%(FullPath)').Contains('\net45\'))) " KeepDuplicates="false">
-        <TranslationFile>$(LocalizationWorkDirectory)\{Lang}\15\%(Filename)%(Extension).lcl</TranslationFile>    <!--Required: translation file-->
+        <TranslationFile>$(LocalizationWorkDirectory)\{Lang}\15\%(Filename)%(Extension).lcl</TranslationFile>
+        <!--Required: translation file-->
```

Comment:
I interpret the comment before this PR to apply to the line for `lcl` files (`<TranslationFile>`), whereas after your change to apply to the `lci` files (`LciCommentFiles>`)

If you want the comment on a separate line, I'd put it before the `<TranslationFile>` element

Reply:
I think all I did was run the default formatter for the file. Let me just undo it since I have no clue what the context was for the comment.